pages/linux/fc.md

```diff
@@ -0,0 +1,15 @@
+# fc
+
+> Open the most recent command and edit it.
+
+- Open normally:
```

Comment:
"normally" isn't super clear on what it does. Maybe we can mention - "Open the most recent command in the default system editor" ?

Reply:
Saying `"Open the most recent command and edit it in the default editor"` seems redundant directly below the title `"Open the most recent command and edit it."`

How does just `"Open with the default system editor"` sound?

Reply:
That sounds great.